libs/data-mapper/src/lib/components/wholeView/WholeView.tsx

```diff
@@ -92,6 +90,7 @@ const WholeOverviewReactFlowWrapper = () => {
       style={reactFlowStyle}
       fitViewOptions={reactFlowFitViewOptions}
       fitView
+      minZoom={0.05}
```

Comment:
Want to see your entire 70mb schema? Well that's probably impossible period, but at least now you can get a lot closer!